### ISSUE TITLE
Feedback: more examples for creating problems?

### ISSUE THREAD

**Issue by sheim** [esa/pagmo#168]
Hi,

I'm having trouble understanding how to write discrete problems. In general I think adding more examples would go a long way to making PyGMO more accessible and get more people to use it.  
For example, an example creating the tsp problem from problem.base would be very helpful.

Also, I've tried to run the built-in tsp problem, and most algorithms I tried complained that they are not suitable for this problem. Listing in the [problem description](http://pagmo.sourceforge.net/pygmo/documentation/problems.html#PyGMO.problem.PyGMO.problem.tsp.tsp.__init__) the algorithms (or at least 1 good algorithm) for solving it helps clarity imho.

**Comment by darioizzo** [comment]
True, the TSP was alate addition and the whole PaGMO project is currently being overhauled hence the seemingly lack of activity in these pages.
